### PR TITLE
Adding StopSpeaking() to TextToSpeechManager

### PR DESCRIPTION
**Pull request by keveleigh**
Requested here (and I think this is useful!): https://forums.hololens.com/discussion/3107/how-do-i-stop-the-tts-from-speaking-in-the-holotoolkit?

Checks to see if the AudioSource is playing and, if it is, stops it. This allows the user to cancel TTS playback mid-stream, without having to keep track of the AudioSource outside TextToSpeechManager.

### CODE REVIEW

**Comment by NeerajW** [comment]
Is there a test scene where you can call this function?

**Comment by keveleigh** [comment]
You bet 😜 i forgot to add it before

**Comment by keveleigh** [comment]
Fix is in

File: Assets/HoloToolkit/Utilities/Scripts/TextToSpeechManager.cs [hunk 1]
```diff
@@ -383,6 +383,33 @@ public void SpeakText(string text)
...
+
+            return false;
+        }
+
+        /// <summary>
+        /// Stops text-to-speech playback.
+        /// </summary>
+        public void StopSpeaking()
+        {
+            if (audioSource != null && audioSource.isPlaying)
```
Comment (jwittner) [inline Assets/HoloToolkit/Utilities/Scripts/TextToSpeechManager.cs]: Nit: should this be `if(IsSpeaking())`

**Pull request merged**